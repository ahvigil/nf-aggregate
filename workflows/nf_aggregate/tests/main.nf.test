nextflow_pipeline {

    name "Test Workflow main.nf"
    script "../main.nf"

    tag "pipeline"
    tag "pipeline_nf-aggregate"

    test("Should run without failures") {

        when {
            params {
                outdir = "results"
            }
        }

        then {
            assert workflow.success
        }

    }

}
