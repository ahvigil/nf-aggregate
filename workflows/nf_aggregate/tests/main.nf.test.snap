{
    "Parameters: default": {
        "content": [
            {
                "0": [
                    
                ],
                "1": [
                    
                ],
                "multiqc_report": [
                    
                ],
                "versions": [
                    
                ]
            }
        ],
        "timestamp": "2023-12-04T10:38:26.10265603"
    },
    "Should run without failures": {
        "content": [
            [
                "versions.yml:md5,1294de7c9d8f7ce29bbb3a638ac1d108",
                "versions.yml:md5,43372d67fa3f029a02cc64a571d88e13"
            ]
        ],
        "timestamp": "2023-12-04T15:47:00.36029546"
    }
}